modules/tripleliftBidAdapter.js

```diff
@@ -249,6 +252,9 @@ function _getORTBVideo(bidRequest) {
       logMessage(`video.placement value of ${video.placement} is invalid for outstream context. Setting placement to 3`)
       video.placement = 3
     }
+    if (!video.plcmt) {
+      video.plcmt = 4;
```

Comment:
prebid already has logic in place doing something similar

Reply:
to support plcmt, pick up what you find in the request, this logic to infer it is not necessary. That's true for the old value as well.

Reply:
in current versions? or in the upcoming v9? Because currently if `video.plcmt` isn't set, it will not be added to the bid request

Reply:
since https://github.com/prebid/Prebid.js/pull/10438